File: patterns/c-patterns/pattern131.c
// Pattern 131: ...

//   * *
// *   *
//     *
//     *
// * * * * *

#include <stdio.h>

int main()
{
    int n = 5;
    int mid = n / 2 + 1;
    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n; j++)
        {
            if (j == mid || i == n || (i + j == mid && j <= mid))
            {
                printf("* ");
            }
            else
            {
                printf("  ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/javascript-patterns/pattern131.js
// Pattern 131: ...

//   * *
// *   *
//     *
//     *
// * * * * *

let n = 5;
let mid = Math.floor(n / 2) + 1;

for (let i = 1; i <= n; i++) {
    let row = '';
    for (let j = 1; j <= n; j++) {
        if (j === mid || i === n || (i + j === mid && j <= mid)) {
            row += '* ';
        } else {
            row += '  ';
        }
    }
    console.log(row);
}
